src/main/java/lists.java

```diff
@@ -0,0 +1,32 @@
+import java.util.ArrayList;
+import java.util.List;
+
+public class lists {
```

Comment:
I think the first letter of class names need to be capitalised, followed by camel casing.